BUILDING.md

```diff
@@ -209,7 +209,7 @@ Next, let's build `runc`:
 
 ```sh
 cd /go/src/github.com/opencontainers/runc
-make BUILDTAGS='seccomp apparmor selinux' && make install
+make BUILDTAGS='seccomp' && make install
```

Comment:
maybe add: 
```
The "selinux" and "apparmor" buildtags have been removed, and now all runc
builds will have SELinux and AppArmor support enabled. Note that "seccomp"
is still optional (though we very highly recommend you enable it).
```

Reply:
`BUILDTAGS=seccomp` is specified by default, so just `make && make install` should work

Reply:
The default runc BUILDTAGS gets overwritten if passing any BUILDTAGS to `make cri-cni-release`, because it passes the same value through to the runc Makefile if it isn't explicitly set here.

I've been building with `make BUILDTAGS=no_btrfs cri-cni-release` and containerd started failing in my test k8s clusters after pulling this change into master.

```
(in script/setup/install-runc)

HEAD is now at 12644e61 VERSION: release 1.0.0~rc93
make[1]: Entering directory '/tmp/tmp.ZJzc2KtI0A/runc'
go build -trimpath "-mod=vendor" "-buildmode=pie"  -tags "no_btrfs" -ldflags "-X main.gitCommit="12644e614e25b05da6fd08a38ffa0cfe1903fdec" -X main.version=1.0.0-rc93 " -o runc .
                                                   ^^^^^^^^^^^^^^^^
```

Maybe this should be documented somewhere, or if the runc build tags aren't intended to be changed from the default then go back to setting it here again?

Reply:
Hmm... good one; that's because both containerd and runc use the same variable name. Also wondering if `runc` looks at environment variables, or make variables 🤔

I guess we can add the `BUILDTAGS` back here, because we always build with the same options in our script

Reply:
opened https://github.com/containerd/containerd/pull/5184